python/configurations/config.yaml

```diff
@@ -1,11 +1,11 @@
     clean_file:
```

Comment:
Should we rename this file as we have multiple configurations now?

Reply:
Sure! The recent commit reflects this change.